src/one_dragon/base/operation/one_dragon_context.py

```
@@ -297,11 +297,67 @@ def init(self) -> None:
+                    else:
+                        log.warning(f"配置 Git hooks 失败 (返回码 {set_result.returncode}): {set_result.stderr.strip()}")
+                        return
+                    
+                    # 赋予执行权限
+                    pre_push_file = githooks_dir / "pre-push"
+                    if pre_push_file.exists():
+                        try:
+                            pre_push_file.chmod(0o755)
+                        except Exception as pe:
+                            log.warning(f"赋予 pre-push 脚本可执行权限失败: {pe}")
+        except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)